sdk-artifact-sync.py

```diff
@@ -18,10 +18,12 @@
 """
 
 import argparse
+import base64
 import os
 import subprocess
 import sys
 import urllib2
+import xml.etree.ElementTree
```

Comment:
Usually this gets aliased by adding `as ElementTree` so that you don't need to fully qualify it below